spec/types/calculation.md

```diff
@@ -192,6 +192,9 @@ This algorithm takes a calculation `calc` and returns a number or a calculation.
 * Let `arguments` be the result of [simplifying](#simplifying-a-calculationvalue) each
   of `calc`'s arguments.
 
+* If `calc` is contained within a supports declaration, return a calculation
```

Comment:
It's a bit confusing that this bullet point comes after simplifying `calc`'s arguments, but doesn't use the `arguments` variable. I might move it to the beginning and say "If ..., return `calc` as-is."